File: XMLTest/test_write.py
import xml.etree.ElementTree as ET

# root = ET.Element('data')
# root.append()
# country = ET.SubElement(root,'country', {'name':'Liechtenstein'})
# rank = ET.SubElement(country,'rank')
# rank.text = '1'
# year = ET.SubElement(country,'year')
# year.text = '2008'
# tree=ET.ElementTree(root)
# tree.write("write_py.xml")

if __name__ == "__main__":
    root = ET.Element('DBUSER')
    tree = ET.ElementTree(root)
    root.text = "" 
    root.text += "test words"
    test2 = ET.Element("test2")
    root.append(test2)
    tree.write("write_py.xml")

File: XMLTest/test_write.cpp
#include <cstdlib> 
#include <cstring> 
#include <ctime> 
#include <string>
#include <iostream>
#include <fstream>
#include <filesystem>
#include <vector>
#include "tinyxml2.h"

using namespace std;

int main() {
	XMLDocument doc;
	XMLElement* root=doc.NewElement("DBUSER");
	doc.InsertEndChild(root);
	XMLText *test1 = doc.NewText("test words");
	root->InsertEndChild(test1);
	XMLElement *test2 = doc.NewElement("test2");
	root->InsertEndChild(test2);
	doc.SaveFile("write_cpp.xml");
	return 0;
}